packages/wrangler/src/index.tsx

```diff
@@ -491,7 +491,7 @@ export async function main(argv: string[]): Promise<void> {
       render(
         <Dev
           name={args.name || config.name}
-          entry={filename}
+          entry={path.relative(process.cwd(), filename)}
```

Comment:
What happens if `wrangler` is run from a subfolder?
Should we instead, be computing paths relative to the `wrangler.toml` file? I.e. the project root?

Reply:
> What happens if wrangler is run from a subfolder?
Let me test that.

Reply:
Works with the current fix. Config pathing from `wrangler.toml` doesn't sounds like a bad idea. Similar to how tsconfig does it. 
<img width="849" alt="Screen Shot 2021-12-28 at 9 17 28 AM" src="https://user-images.githubusercontent.com/27247160/147581833-645aff0a-6fab-4c67-869d-fe79bb72bab5.png">

Reply:
Oh I will move the script file to root and run wrangler from the subfolder... one sec

Reply:
Still working with relative pathing for the script. 
<img width="265" alt="Screen Shot 2021-12-28 at 9 33 26 AM" src="https://user-images.githubusercontent.com/27247160/147582125-34ea242f-4f2b-42b1-bdb6-98fe53e00e65.png">

Reply:
Ah yes because the path passed to the CLI as the entry file is always relative to the current working directory.